utils/config_manager.py

```
@@ -1838,6 +1853,25 @@ def save_workshop_path(self, workshop_path):
+        """
+        if self._user_workshop_folder_persisted:
+            return
+        if not workshop_path or not os.path.isdir(workshop_path):
+            return
+        self._user_workshop_folder_persisted = True
+        try:
+            config = self.load_workshop_config()
+            config["user_workshop_folder"] = workshop_path
+            self.save_workshop_config(config)
+            logger.info(f"已持久化Steam创意工坊路径到配置文件: {workshop_path}")
+        except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)